jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/ConstructorDefinition.java

```diff
@@ -0,0 +1,29 @@
+package com.fasterxml.jackson.jr.ob.impl;
```

Comment:
Um, sorry, but this looks unnecessary complicated: instead of `Map`, suppliers etc, let's just stick to plain old typed fields. Map adds overhead while not improving readability.